### DISCUSSION TITLE
Is it good decision to build currently large scale web application using NextJS it's not stable though!

### DISCUSSION THREAD

**Discussion by viva-lab** [community/community#63648] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

Is it good decision to build currently large scale web application using NextJS it's not stable though!

**Comment by joy-ahmed** [comment]
You can ask yourself what it is, cause every problem has a solution, NextJS is changing their ecosystem and updating their product to improve or enhance development process as well as they have a very active community where you can get any kinda help, well rest of the things up to you

**Comment by naveedalirehmani** [comment]
The new feature of route intercepting with parallel routes in next.js caused me some trouble. Many other developers are experiencing the same issues and some have found solutions to this by grouping each segment into a separate group, which shouldn't be the case.
Personally, I have had a lot of issues with next.js, as described above :D

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
